package/MDAnalysis/auxiliary/base.py

```diff
@@ -43,11 +43,12 @@
 from six.moves import range
 
 import os
-import numpy as np
+import numbers
 import math
 import warnings
 
 from ..lib.util import asiterable, anyopen
+import numpy as np
```

Comment:
the numpy import should still be independent from the mdanalysis internal imports